dbclient/dbclient.py

```diff
@@ -40,6 +54,14 @@ def __init__(self, configs):
             os.environ['CURL_CA_BUNDLE'] = ""
         os.makedirs(self._export_dir, exist_ok=True)
 
+    def _update_url_and_token(self, url, token):
+        self._url = url_validation(url)
```

Comment:
I'm not sure in what cases the URL will be changed, so I'm not sure if it is legitimate. Functional-wise, it won't work if URL is changed but not updated here.

Reply:
The URL shouldn't change in the middle of the migration, so instead of updating the url, we should compare it with the previously set url, log error and terminate. WDYT ?

Reply:
I agree that I don't see a use case where updating the URL would be necessary.

Reply:
Removed updating URL logic.